core/integration/features/margin/6202-release-excess-on-amend.feature

```diff
@@ -57,23 +60,28 @@ Feature: test margin during amending orders
     # check the requried balances
     And the parties should have the following account balances:
       | party  | asset | market id | margin | general  | bond  |
-      | party1 | USD   | ETH/MAR22 | 19218  | 99980782 | 0     |
-      | party4 | USD   | ETH/MAR22 | 85366  | 99914634 | 0     |
+      | party1 | USD   | ETH/MAR22 | 36514  | 99963486 | 0     |
+      | party4 | USD   | ETH/MAR22 | 179268 | 99820732 | 0     |
+      # | party1 | USD   | ETH/MAR22 | 19218  | 99980782 | 0     |
+      # | party4 | USD   | ETH/MAR22 | 85366  | 99914634 | 0     |
     #check the margin levels
     Then the parties should have the following margin levels:
       | party  | market id | maintenance | search | initial | release |
-      | party1 | ETH/MAR22 | 16015       | 17616  | 19218   | 22421   |
-      | party4 | ETH/MAR22 | 71139       | 78252  | 85366   | 99594   |
+      | party1 | ETH/MAR22 | 30429       | 33471  | 36514   | 79114   |
+      | party4 | ETH/MAR22 | 149391      | 164329 | 179268  | 209146  |
+      # | party1 | ETH/MAR22 | 16015       | 17616  | 19218   | 22421   |
+      # | party4 | ETH/MAR22 | 71139       | 78252  | 85366   | 99594   |
```

Comment:
I think the margin for party4 should be the old value: 71139, the same as margin for party1